cmd/zetaclientd/keygen_tss.go

```diff
@@ -103,13 +106,26 @@ func GenerateTss(logger zerolog.Logger, cfg *config.Config, zetaBridge *mc.ZetaC
 }
 
 func keygenTss(cfg *config.Config, tss *mc.TSS, keygenLogger zerolog.Logger) error {
-
 	keygenLogger.Info().Msgf("Keygen at blocknum %d , TSS signers %s ", cfg.Keygen.BlockNumber, cfg.Keygen.GranteePubkeys)
 	var req keygen.Request
 	req = keygen.NewRequest(cfg.Keygen.GranteePubkeys, cfg.Keygen.BlockNumber, "0.14.0")
 	res, err := tss.Server.Keygen(req)
 	if res.Status != tsscommon.Success || res.PubKey == "" {
 		keygenLogger.Error().Msgf("keygen fail: reason %s blame nodes %s", res.Blame.FailReason, res.Blame.BlameNodes)
+		// Need to broadcast keygen blame result here
+		digest, err := digestReq(req)
+		if err != nil {
+			return err
+		}
+		zetaHash, err := tss.CoreBridge.PostBlameData(&res.Blame, common.ZetaChain().ChainId, digest) //common.GetChainFromChainID(common.ZetaChain().ChainId)
```

Comment:
What does the comment mean?

Reply:
Forgot to remove the previous implementation.